lib/charms/loki_k8s/v1/loki_push_api.py

```diff
@@ -20,6 +20,9 @@
 send telemetry, such as logs, to Loki through a Log Proxy by implementing the consumer side of the
 `loki_push_api` relation interface.
 
+- 'LogForwarder': This object can be used by any Charmed Operator which needs to send its
```

Comment:
```suggestion
- 'LogForwarder': This object can be used by any Charmed Operator which wants any stdout emitted by a workload container to be captured and automatically forwarded to Loki. 
```

Reply:
as it's written, it gives the impression we're talking about the charm process' own stdout